docs/about.md

```diff
@@ -0,0 +1,70 @@
+# Cloud automation for Account Configuration
+
+This deployable architecture prepares your IBM Cloud account with the necessary configurations to ensure a secure and organized environment for your architecture.
+
+This deployable architecture is **typically consumed as a dependency** by other IBM Cloud deployable architectures published in the catalog, providing the foundational account setup they require.
+
+You can choose from two variations when deploying:
+
+* **Resource groups only**: Creates a structured set of IBM Cloud resource groups to help organize cloud resources by function.
+* **Resource groups with Account Settings**: In addition to resource groups, configures account-level settings for access control, security, and compliance alignment.
+
+---
+
+## Resource Groups – Organizing Your Cloud Resources
+
+IBM Cloud uses Resource Groups to help organize and manage your cloud resources, similar to folders on your computer.
```

Comment:
I would just add a link to ibm cloud doc on concept such as resource group, account settings, etc